src/Index/Model/IndexDocument.php

```diff
@@ -14,25 +14,25 @@
 
 namespace Gally\Index\Model;
 
+use ApiPlatform\Core\Action\NotFoundAction;
 use ApiPlatform\Core\Annotation\ApiProperty;
 use ApiPlatform\Core\Annotation\ApiResource;
 use Gally\User\Constant\Role;
 
 #[
     ApiResource(
         collectionOperations: [
-            'get',
             'post' => ['security' => "is_granted('" . Role::ROLE_ADMIN . "')"],
         ],
         graphql: [
-            'item_query',
-            'collection_query',
             'create' => ['security' => "is_granted('" . Role::ROLE_ADMIN . "')"],
-            'delete' => ['security' => "is_granted('" . Role::ROLE_ADMIN . "')"],
         ],
         itemOperations: [
-            'get',
-            'delete' => ['security' => "is_granted('" . Role::ROLE_ADMIN . "')"],
+            'get' => [
+                'controller' => NotFoundAction::class,
+                'read' => false,
+                'output' => false,
+            ],
         ],
         paginationEnabled: false,
     ),
```

Comment:
Isn't this a totally unrelated subject ?

Reply:
No, see the excel file in the ticket https://elasticsuite.myjetbrains.com/youtrack/issue/ESPP-649/ , especailly the colors :stuck_out_tongue:

Reply:
Not seeing any excel file included in https://elasticsuite.myjetbrains.com/youtrack/issue/ESPP-649/ :man_shrugging: